strapi/strapi
# feat(translations): add several items in zh-Hans

### What does it do?

Add several translations for zh-Hans.

### Why is it needed?

Current pkg lack of these items.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

eg:

```json
  "app.components.LeftMenu.navbrand.title": "Strapi 仪表盘",
  "app.components.LeftMenu.navbrand.workplace": "工作空间",
  "containers.SettingPage.editSettings.description": "拖放字段来构建布局",
  "containers.SettingPage.add.relational-field": "插入另一个关系字段",
  "content-type-builder.menu.section.models.name": "集合类型",
  "content-type-builder.menu.section.single-types.name": "单一类型",
  "content-type-builder.menu.section.components.name": "组件",
  "content-type-builder.button.model.create": "创建一个新的集合类型",
  "content-type-builder.button.single-types.create": "创新一个新的单一类型",
  "content-type-builder.button.component.create": "创建一个新的组件",
  "content-type-builder.listView.headerLayout.description": "定义内容的数据结构",
  "content-type-builder.form.button.add.field.to.collectionType": "为这个集合类型添加另一个字段",
  "content-type-builder.form.button.add.field.to.singleType": "为这个单一类型添加另一个字段",
  "components.FilterOptions.FILTER_TYPES.$neq": "不等于",
  "global.save": "保存",
  "global.back": "返回",
  "global.documentation": "文档",
  "global.name": "名称",
  "global.type": "类型",
  "global.delete-target": "删除 {target}",
```
